### PR TITLE
Readme updated with new special token addition instructions

### PR DESCRIPTION
**Pull request by floccinauc**

### CODE REVIEW

File: fusedrug/data/protein/tokenizer/build_protein_tokenizer_simple.py [hunk 1]
```diff
@@ -111,8 +111,8 @@ def main(vocab_name: str, output_tokenizer_json_file: str, unknown_token: str):
 
     build_simple_vocab_protein_tokenizer(
         vocab=vocab_name,
-        unknown_token=unknown_token,  #'<UNK>',
-        save_to_json_file=output_tokenizer_json_file,
+        unknown_token=unknown_token,
+        save_to_json_file=output_tokenizer_json_file,  #'<UNK>',
```
Comment (mosheraboh) [inline fusedrug/data/protein/tokenizer/build_protein_tokenizer_simple.py:115]: the unknown comment should be in line 114

File: fusedrug/data/tokenizer/modulartokenizer/README.md [hunk 2]
```diff
@@ -67,22 +68,34 @@ Saves the underlying adjusted tokenizers as jsons.
 Returns the input token's corresponding ID.
 ## Use example
 ### Creation:
-An example of creation of a new modular tokenizer from a word-level AA sequence tokenizer and a BPE SMILES tokenizer is found here: [ModularTokenizer creation](https://github.com/BiomedSciAI/fuse-drug/blob/a1b9564eb54b9fe39890645fb5378c13aedde6fb/fusedrug/data/tokenizer/modulartokenizer/test_multi_tokenizer_creation.py#L107)
+An example of creation of a new modular tokenizer from a word-level AA sequence tokenizer and a BPE SMILES tokenizer is found here: [ModularTokenizer creation](https://github.com/BiomedSciAI/fuse-drug/blob/5aaef572df7679606ec8a8a181013e54448f5ad5/fusedrug/data/tokenizer/modulartokenizer/test_multi_tokenizer_creation.py#L238)
```
Comment (mosheraboh) [inline fusedrug/data/tokenizer/modulartokenizer/README.md:71]: use relative pointers instead, so it will point to the latest

**Review by mosheraboh — CHANGES_REQUESTED** [review]
Thanks @floccinauc. Looks good.
See minors inline.

File: fusedrug/data/protein/tokenizer/build_protein_tokenizer_simple.py (hunk 1 above)
Comment (floccinauc) [reply fusedrug/data/protein/tokenizer/build_protein_tokenizer_simple.py:115]: I don't know why this change was attributed to me - this isn't my code, and I've never even looked at it. It's not fully typed, so it won't pass mypy tests, so I can't even merge it properly...

Comment (floccinauc) [reply fusedrug/data/protein/tokenizer/build_protein_tokenizer_simple.py]: Changed it anyway. Weird.

**Review by mosheraboh — APPROVED** [review]

**Pull request merged**